src/datasources/fault-ds/partials/query.editor.html

```diff
@@ -11,8 +11,16 @@
 
     </onms-query>
 
-    <div class="gf-form" style="padding-top: 10px">
-        <button class="btn btn-inverse gf-form-btn" ng-click="ctrl.clearRestrictions()" ng-show="ctrl.showClearRestrictions()">
+    <div class="gf-form" style="padding-top: 15px">
+        <gf-form-switch class="gf-form"
+                        label="Featured attributes"
+                        tooltip="Used to toggle whether to show a list of featured attributes or all attributes when selecting an attribute"
```

Comment:
I'd rephrase this a little shorter as:
Toggles whether featured attributes or all attributes are shown.